app/containers/InterventionDetailsPage/containers/ParticipantInviter/ParticipantListView.tsx

```diff
@@ -26,7 +23,7 @@ export type Props = {
   invitingPossible: boolean;
   copyingInvitationLinkPossible: boolean;
   creatingPredefinedParticipantsPossible: boolean;
-  sessionOptions: SelectOptionTypeExtended<string>[];
+  sessionOptions: (SelectOption<string> & { type: string })[];
```

Comment:
You don't need that here
```suggestion
  sessionOptions: SelectOption<string>[];
```